src/pages/Signup/index.jsx

```diff
@@ -298,9 +298,9 @@ const SignupScreen = () => {
             handleSignInWithFacebook={(e) =>
               signInWithOAuth(e, enqueueSnackbar, navigate, false)
             }
-            have_acct_question={"Already have an account?"}
+            have_acct_question={"Don't have an account?"}
```

Comment:
I think perhaps this doesn't make sense, as we'd definitely want to sign in as an alternative in signup page. Also we'd want to navigate to signin from signup.